app/DoctrineMigrations/Version20180314160032.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types = 1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180314160032 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE TABLE delivery_order (order_id INT NOT NULL, user_id INT NOT NULL, PRIMARY KEY(order_id, user_id))');
```

Comment:
Nice to separate delivery/eshop order into tables

Reply:
This is actually a workaround 🙂
In the base Sylius Order component, an Order is not linked to anything. 
However, it totally makes sense to link an Order with a Customer. 

When #155 is fixed, we will be able to extend Sylius models, for example to implement [`CustomerAwareInterface`](https://github.com/Sylius/Sylius/blob/f2042d449c5ae3b68df145910cb42fae6c90d254/src/Sylius/Component/Customer/Model/CustomerAwareInterface.php)